recipe/conda_build_config.yaml

```diff
@@ -36,14 +36,17 @@ m2w64_fortran_compiler:        # [win]
 
 cuda_compiler:
   - None
-  - nvcc                       # [(linux64 or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - nvcc                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 cuda_compiler_version:
   - None
-  - 11.2                       # [(linux64 or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 11.2                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 cuda_compiler_version_min:
   - None                       # [osx]
   - 11.2                       # [linux or win64]
 
+arm_variant_type:              # [aarch64]
+  - sbsa                       # [aarch64]
```

Comment:
Should this be limited to where `CF_CUDA_ENABLED` is `TRUE` or does this key only get used in those situations anyway?

Does this impact any non-CUDA packages?

Reply:
So far this is mainly used with CUDA

Maybe it makes sense if there are CPU-only packages that do particular things on different ARM variants. Not aware of any such use cases today. Though given the nature of ARM, this doesn't seem too hard to imagine